### DISCUSSION TITLE
commercial use questions

### DISCUSSION THREAD

**Discussion by Eliasjk17** [lllyasviel/Fooocus#3362] [Q&A] [answered]
First of all, I apologize if this is not understood perfectly, I am using a translator.I also apologize for never using this website and between that and the language I had problems recognizing if this issue belongs here or in another section.

I saw that the approval sign where it says: I use commercials, but due to the complexity of the technical language and that I do not speak English, it is not completely clear to me.

My case is the following, I want to use Fooocus to generate a personal digital marketing project, I want to create an influencer on a social network like Instagram, without any type of monetization, so far I don't think there is any problem from what I could understand, But if this project and that Influencer generated by AI range fame were to stop being a personal project and turn it into a commercial project, from which the main monetary income would be received from pages like OnlyFans, Patreon or perhaps they are more familiar with pages like FanCentro Due to the cultural difference, I still explain it briefly, they are social network pages where people pay a subscription to your profile to see XXX content, Fooocus could be used for commercial uses on this type of pages/social networks (Everything would be about a woman created with AI would not be real)

Thanks and greetings

**Comment by mashb1t** [comment, marked as answer]
What's the question, if you can use it commercially?

Already answered in 
- https://github.com/lllyasviel/Fooocus/discussions/3029
- https://github.com/lllyasviel/Fooocus/discussions/2688
- https://github.com/lllyasviel/Fooocus/discussions/1957

**Reply by mashb1t** [reply]
also see https://github.com/lllyasviel/Fooocus/discussions/1957#discussioncomment-8283320

**Reply by Eliasjk17** [reply]
Rather, if I had to summarize everything in a single serious question, can you use the XXX images generated with Fooocus for XXX commercial purposes?

**Reply by Eliasjk17** [reply]
> Véase también [#1957 (comentario)](https://github.com/lllyasviel/Fooocus/discussions/1957#discussioncomment-8283320)
I know it must be hard to always answer the same questions, but I'm really having problems with the language, I'm sorry, could you give me a simple answer like yes or no? translators don't work very well with complex language or long texts

**Reply by mashb1t** [reply]
yes, with exceptions and without Fooocus V2 style. You may also not redistrubute the code with another license. I am no lawyer, please consult one if you're uncertain.

See https://github.com/lllyasviel/Fooocus/blob/main/LICENSE and excerpt:
> For non-commercial use only - if you use this in commercial project then
probably it has some intellectual property issues.

**Reply by Eliasjk17** [reply]
> Sí, con excepciones y sin el estilo Fooocus V2

So if I understood correctly, as long as you don't use the Fooocus V2 style you shouldn't have problems with a commercial XXX one, right? And as for distributing it, for now it is not my intention, but just in case I consider it later, I understand that it can be done as long as I do not register it under another commercial license, that is, if I, for example, do it If I sold to a person and gave them the program through a pen drive or Google drive, there would be no problem, right?

**Reply by mashb1t** [reply]
You can only distribute the code with GPL v3 (open source license) again, no matter how you distribute it. The other person has the right to publish whatever you did if it's based on any program using GPL v3, which Fooocus is.

Please do not only check the license of Fooocus as the tool but also the models you intend to use as they can differ from base SDXL and can have proprietary licenses.

**Reply by Eliasjk17** [reply]
How can I verify if a specific model is allowed for commercial use? In principle I will only use the models, Fooocus Photograph, Photo Iphone Photographic and Cinematic Diva, the rest of the image retouching will be done with Photoshop.

**Reply by mashb1t** [reply]
You have to check their individual licenses, e.g. on Civitai or Huggingface.

**Reply by Eliasjk17** [reply]
> Debes comprobar sus licencias individuales, por ejemplo en Civitai o Huggingface.

But how do I do this exactly? Because I searched in civitai for the models by the name of Cinematic Diva, Fooocus Photograph, Fooocus Negative and Photo iPhone Photographic and these models do not appear or at least not by the same name, which makes it impossible for me to verify them. The same thing happens to me on Huggingface, in the search for models section they do not appear or they do not appear with the same name.

**Reply by mashb1t** [reply]
All of these are styles, so prompt only adjustments, which do not require any external model.
Whenever you use a LoRA, model, etc. which requires an external file to be downloaded and installed you have to check their license.

**Reply by Eliasjk17** [reply]
> Todos estos son estilos, por lo que solo solicitan ajustes, que no requieren ningún modelo externo.Siempre que utilice un LoRA, modelo, etcétera que requiera que se descargue e instale un archivo externo, debe verificar su licencia.

Hoooo... okay I think I understand it, I will only need to verify these models if they are downloaded and implemented in Fooocus by me, but on the other hand if I only use the ones from version 2.5.0 without adding any extra models I will be able to give them commercial use ( in this case I use commercial XXX) as long as I don't use the Fooocus V2 style, correct?

**Reply by mashb1t** [reply]
styles: yes
models/loras: please double check on CivitAI/Huggingface.

**Reply by Eliasjk17** [reply]
I understand that everything that is implemented in Fooocus 2.5.0 out of the box except for Fooocus V2 are styles, correct?

**Reply by mashb1t** [reply]
no, this is incorrect. Each preset, also the default one, downloads a set of models and LoRAs, which you also have to check.

**Reply by Eliasjk17** [reply]
> Cinematic Diva, Fooocus Photograph, Fooocus Negative and Photo iPhone Photographic

I understand, I will still only use these 4: Cinematic Diva, Fooocus Photograph, Fooocus Negative and Photo iPhone Photographic. Can I use those 4 for the issue I mentioned about using them for XXX commercial content?

**Reply by mashb1t** [reply]
yes, but also check the other things. You need a model to create images.

**Reply by Eliasjk17** [reply]
> yes, but also check the other things. You need a model to create images.

Ok, sorry, I know what you mean, I didn't have those windows explored yet, the model it uses is: RealisticStockPhoto_v20.safetensors, and the Lora is: SDXL_FILM_PHOTOGRAPHY_STYLE_V1.safetensors

Now could you guide me step by step to verifity if I can use them for what I want please? I'm very sorry to be giving you so much work, or since I probably don't use any of the other two, could you look at it and tell me if these can be used, please?
